src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -15,6 +15,13 @@
 from azure.mgmt.storage import StorageManagementClient
 from azure.storage.models import ResourceTypes, Services
 from azure.storage.blob.models import ContainerPermissions
+from azure.storage.table.models import TablePermissions
+from azure.storage.queue.models import QueuePermissions
+
+class IgnoreAction(argparse.Action): # pylint: disable=too-few-public-methods
+    def __call__(self, parser, namespace, values, option_string=None):
+        raise argparse.ArgumentError(None, 'unrecognized argument: {} {}'.format(
+            option_string, values or ''))
```

Comment:
Just like what you've done for values, you might need "or ''" for option_string as well otherwise you could end up printing None.

>>> print('{}{}'.format(None, 'helloworld'))
Nonehelloworld